WordPress/src/main/java/org/wordpress/android/models/MediaUploadState.java

```diff
@@ -18,4 +22,22 @@ public static MediaUploadState fromString(String strState) {
         }
         return UPLOADED;
     }
+
+    public static String getUIString(Context context, MediaUploadState state) {
```

Comment:
Yes, thank you!! I remember we had a name for it, but couldn't think of an example to check it out :(